src/OSPSuite.Core/Chart/Simulations/SimulationPredictedVsObservedChart.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace OSPSuite.Core.Chart.Simulations
+{
+   public class SimulationPredictedVsObservedChart : PredictedVsObservedChart
+   {
+      public SimulationPredictedVsObservedChart()
+      {
+         ChartSettings.LegendPosition = LegendPositions.BottomInside;
```

Comment:
@georgeDaskalakis Shouldn't that be the default for PI as well?

Reply:
true, moved this to  PredictedVsObservedChart